phpBB/styles/prosilver/template/attachment.html

```diff
@@ -8,14 +8,14 @@
 
 		<!-- IF _file.S_THUMBNAIL -->
 		<dl class="thumbnail">
-			<dt><a href="{_file.U_DOWNLOAD_LINK}"><img src="{_file.THUMB_IMAGE}" class="postimage" alt="{_file.DOWNLOAD_NAME}" title="{_file.DOWNLOAD_NAME} ({_file.FILESIZE} {_file.SIZE_LANG}) {_file.L_DOWNLOAD_COUNT}" /></a></dt>
+			<dt><a href="{_file.U_DOWNLOAD_LINK}"><img src="{_file.THUMB_IMAGE}" class="postimage" alt="{% if _file.COMMENT %}{{ _file.COMMENT }}{% else %}{{ _file.DOWNLOAD_NAME }}{% endif %}" title="{_file.DOWNLOAD_NAME} ({_file.FILESIZE} {_file.SIZE_LANG}) {_file.L_DOWNLOAD_COUNT}" /></a></dt>
```

Comment:
Alternatively you can use ternary syntax:
`{{ _file.COMMENT ?: _file.DOWNLOAD_NAME }}`

Reply:
I believe Hanakin is not too fond of that though.

Reply:
What happens if there is a quote in the name or comment? Isn't there something else needed here for escaping?

Reply:
@DavidIQ I will test it and amend the code if necessary.

Reply:
@VSEphpbb Yes but the main reason for this is code's readability, in any case let's see what @hanakin says about that.

Reply:
@DavidIQ it is ok.

![immagine](https://user-images.githubusercontent.com/480857/62892314-484fe000-bd48-11e9-9931-55c35437ae9f.png)

Reply:
That just looks like it's rendering out the quote inside of the alt property making the markup invalid.

Reply:
Hmm, I commented with 

![immagine](https://user-images.githubusercontent.com/480857/62893233-5d2d7300-bd4a-11e9-87fb-b7cb10891d5c.png)

Reply:
Needs to use an escape, like `_file.COMMENT|e('html')`

Reply:
Just on the comment side?

Reply:
yes. File download name's shouldn't have html chars in them

Reply:
How the screen reader will "read" this?

![immagine](https://user-images.githubusercontent.com/480857/62893443-f066a880-bd4a-11e9-9473-9e9ab619cfc6.png)

Reply:
That's proper HTML markup now.

Reply:
Ok

Reply:
Please no ternary in template files